### PR TITLE
Jodconverter 4.2.0

### PR DESCRIPTION
**Pull request by mmoqui**
In order to test this PR, you need first:

- build the PR https://github.com/Silverpeas/silverpeas-dependencies-bom/pull/7
- build Silverpeas-Project in 1.1-SNAPSHOT version
- build this PR
- build Silverpeas-Components with its project parent, Silverpeas-Project, in 1.1-SNAPSHOT version
- build Silverpeas-Assembly with its project parent, Silverpeas-Project, in 1.1-SNAPSHOT.

Once the PR for silverpeas-dependencies-bom merged, a build version of both silverpeas-dependencies-bom, silverpeas-test-dependencies-bom and Silverpeas-Project projects will be built the CI. Then, once, this PR merged, don't forget to launch the build version of  Silverpeas (it will update the parent POM version of all of the projects that made up Silverpeas).

### CODE REVIEW

File: core-library/src/main/java/org/silverpeas/core/contribution/converter/DocumentFormat.java [hunk 1]
```diff
@@ -49,7 +49,11 @@
   /**
    * The HTML format.
    */
-  html(MimeTypes.HTML_MIME_TYPE);
+  html(MimeTypes.HTML_MIME_TYPE),
+  /**
+   * The SQL format.
+   */
+  sql(MimeTypes.PLAIN_TEXT_MIME_TYPE);
```
Comment (silveruser) [inline core-library/src/main/java/org/silverpeas/core/contribution/converter/DocumentFormat.java]: CRITICAL Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [rule](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS00115)

**Comment by silveruser** [comment]
SonarQube analysis reported 1 issue
* ![CRITICAL][CRITICAL] 1 critical

Watch the comments in this conversation to review them.

[CRITICAL]: https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL'

**Comment by silveruser** [comment]
SonarQube analysis reported 1 issue
* ![INFO][INFO] 1 info

Watch the comments in this conversation to review them.

[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'

**Comment by SilverYoCha** [comment]
Closed automatically.
FYI, this PR has also been merged (but not detected by GitHub).

**Pull request closed without merging**